flowsa/__init__.py

```diff
@@ -48,20 +42,8 @@ def getFlowBySector(methodname):
     :param methodname: string, Name of an available method for the given class
     :return: dataframe in flow by sector format
     """
-    fbs = pd.DataFrame()
-    # first try reading parquet from your local repo
-    try:
-        log.info('Loading ' + methodname +' parquet from local repository')
-        fbs = pd.read_parquet(fbsoutputpath + methodname + ".parquet")
-    except (OSError, FileNotFoundError):
-        # if parquet does not exist in local repo, read file from Data Commons
-        try:
-            log.info(methodname + ' parquet not found in local repo, loading from Data Commons')
-            fbs = pd.read_parquet('https://edap-ord-data-commons.s3.amazonaws.com/flowsa/FlowBySector/' +
-                                  methodname + ".parquet")
-        except FileNotFoundError:
-            log.error("No parquet file found for datasource " + methodname + " in flowsa or Data Commons")
-
+    fbs_file = "FlowBySector/" + methodname + ".parquet"
+    fbs = load_preprocessed_output(fbs_file, paths)
```

Comment:
Looks like you haven't yet gotten to FBS, as need to assign fbs_meta to pass to load_preprocessed_output

Reply:
Yes, I've only made the changes for fba's as  a starting place